version/policy.go

```diff
@@ -17,7 +17,7 @@
 
 package version
 
-const policyVersion = "v1.5.5"
+const policyVersion = "8.10"
```

Comment:
if cloudbeat branches are always aligned with csp-security-policies branches, as in `8.10` branch in cloudbeat will fetch the `8.10` branch from csp-policies to build the bundle, then we can drop this `policyVersion` variable and always use `CloudbeatVersion`